da.go

```diff
@@ -2,6 +2,9 @@ package da
 
 // DA defines very generic interface for interaction with Data Availability layers.
 type DA interface {
+	// Config returns the max blob size
+	Config() (uint64, error)
```

Comment:
let's call this `MaxBlobSize` instead of `Config`, as the interface is already aware of `Blob` term.

Reply:
This is a tricky question. In the future we might need to return more configurations/parameters.

There are several ways to put this in API.
1. Add specific methods for every parameter (like `MaxBlobSize`)
2. Use Config method that returns what's needed (single value now, a struct later).
3. Use generic method that returns a map with predefined keys/value formats. This adds the possibility to provide some implementation specific params, but on the other hand those params are not portable (go-da user (like rollkit) shouldn't really care about per-implementation params).

Depending on how many params we have, I would lean into separate methods (if not many params) or config struct (in case of many params).

Obviously we can change at any time, so maybe we should start with `MaxBlobSize`.

Reply:
Preference for single methods until we have multiple.

Reply:
Renamed